nvflare/apis/utils/fl_context_utils.py

```diff
@@ -63,7 +63,9 @@ def generate_log_message(fl_ctx: FLContext, msg: str):
 
     peer_ctx = fl_ctx.get_peer_context()
     if peer_ctx:
-        assert isinstance(peer_ctx, FLContext)
+        assert isinstance(peer_ctx, FLContext), "peer_ctx must be an instance of FLContext, but got {}".format(
+            type(peer_ctx)
+        )
```

Comment:
Same here, we need to think if we need this check or not.

If we need: suggest change to if instance raise
If we don't: remove this whole assert statement